Plotter/plot.py

```diff
@@ -9,7 +9,7 @@
 #>>>>Run with --serial option if using py3:
 #   ./plot.py -y 2018 -c mutau --serial
 
-from config.samples import *
+from config.samples_v12 import *
```

Comment:
This may affect other people's work, but it's a flaw in the design of the TauFW configuration that should be addressed asap.

Reply:
I think, something needs to be done here so that there is no need to make this changes manually whenever list of samples are changed with era. For 2018, I used _v10 and for 2023 I used _v12. So whenever I changed the era, I used to change this manually.

Reply:
Yes, this needs to be addressed, but falls outside the scope of this PR. Just leave it like this for now.